wordpress-mobile/WordPress-Android
# Unified Login & Sign-Up: Fix issues with Gravatar placeholder

Fixes #12379 and #12380

To test:

1. On the Prologue screen, if the Smart Lock dialog appears, dismiss it.
1. Tap **Continue with WordPress.com**.
1. On the Get Started screen, enter an email address associated with an account with no Gravatar profile picture.
1. Tap **Continue**.
1. On Login Magic Link screen, notice the default Gravatar icon is both _outlined_ and _grey_.
1. Complete the login flow normally.
1. On the Epilogue screen, notice the default Gravatar icon is both _outlined_ and _grey_.

Notes:
- Use a device with API 21 and 28 to test.
- Avoid using an account with no sites to test, as it might make the Post-Signup Interstitial to be shown instead of the Epilogue.

PR submission checklist:
